Request/AbstractRequestBodyParamConverter.php

```diff
@@ -151,21 +155,23 @@ protected function execute(Request $request, ParamConverter $configuration)
     }
 
     /**
-     * @return DeserializationContext
+     * @return DeserializationContext|Context
      */
     protected function getDeserializationContext()
```

Comment:
Yeah probably better to import the 2.0 features to 1.8 when possible and then remove all the deprecations in 2.0.